models/api-key.js

```diff
@@ -182,6 +182,8 @@ const isValidApiSecret = (apiKeyRecord, apiSecret = '') => {
   const isValid = password.compare(apiSecret, apiKeyRecord.hashedApiSecret);
   if (isValid !== true) {
     console.log('The given API Secret is NOT valid for the given API Key record.');
+    const redactedApiSecret = apiSecret.substring(0, 3) + '...[snip]...' + apiSecret.substring(apiSecret.length - 3);
+    console.log(redactedApiSecret);
```

Comment:
This ends up logging a message like this (which you can see during tests):
`tTi...[snip]...4c=`

Note that the specific value I pasted isn't used anywhere, it's just from my running `make test` locally in this repo.